Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:lts-stretch
```

Comment:
this results in a very large image (1.23GB), we might leverage multi-stage docker builds as described in https://github.com/nodejs/docker-node/issues/282